### PR TITLE
	modified:   src/app/options/loadshedding/loadshedding.component.html

### PR DESCRIPTION
**Pull request by Ruandv**
I created a PR so that you can monitor what I am busy with. 

	modified:   src/app/options/loadshedding/loadshedding.component.ts
		- Added the onLicenseChanged event to go get the National Status data so long.
	modified:   src/app/tab/loadshedding/loadshedding.component.html
	modified:   src/app/tab/loadshedding/loadshedding.component.ts
		- Retrieve and display data.
	modified:   src/app/tab/loadshedding/loadshedding.service.ts
		- Added the national status call to the service.
	modified:   src/scss/_loadshedding.scss
		- Styling in progress.

### CODE REVIEW

**Comment by Ruandv** [comment]
@bluecaret when you are ready you can review this PR and if there are any changes I need to make let me know.

**Comment by bluecaret** [comment]
Thank you @Ruandv I'll try to get this reviewed this week, quick glance it looks good though! Thanks again for the work on this!

File: src/app/tab/loadshedding/loadshedding.component.html [hunk 1]
```diff
@@ -1,16 +1,19 @@
...
-</div>
+<div class='loadsheddingContainer'>
+   <div class="suburb">
+    <div class="header">
+      <h2>{{cachedAreaInfo?.data?.info?.name}} </h2>
+      <label class="dim">{{cachedAreaInfo?.data?.info?.region}}</label>
+    </div>
+    <div class="schedules">
+      <ng-container *ngFor="let day of cachedAreaInfo.data?.schedule?.days">
+        <div class="daySchedule" *ngIf="showData(day).length >0">
```
Comment (bluecaret) [inline src/app/tab/loadshedding/loadshedding.component.html:9]: Getting an error when loading a tab that I'm pretty sure is related to this line: 

```
main.js:9070 ERROR TypeError: Cannot read properties of undefined (reading 'length')
    at Object.updateDirectives (main.js:9070:817)
```

Comment (Ruandv) [reply src/app/tab/loadshedding/loadshedding.component.html:9]: I found another bug also. Would have to spend some time sorting it out.

Comment (Ruandv) [reply src/app/tab/loadshedding/loadshedding.component.html:9]: Sorted.

Comment (bluecaret) [reply src/app/tab/loadshedding/loadshedding.component.html:9]: There was still a bug here, looks like it just needs this ngIf to be changed to `*ngIf="showData(day) && showData(day).length >0"`

I've got some time today to work on this though so I'll take what you have and make that fix and a couple small styling things and get this going. Thank you so much for the work on this!

**Pull request merged**